torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -272,8 +272,8 @@ def affine_image_tensor(
     if image.numel() == 0:
         return image
 
-    num_channels, height, width = image.shape[-3:]
-    extra_dims = image.shape[:-3]
+    shape = image.shape
+    num_channels, height, width = shape[-3:]
```

Comment:
Minor nit to align idioms across the code-base. See ref:
https://github.com/pytorch/vision/blob/70faba91d1c79f689ac6ebd30ad0c4be62690196/torchvision/prototype/transforms/functional/_geometry.py#L37

Originally I wanted to do something like `*extra_dims, num_channels, height, width` but that's not JIT-scriptable. So I opted for keeping the whole original shape as we do on other parts of the code-base.